givitsite/coordinate/views.py

```diff
@@ -1,3 +1,67 @@
-from django.shortcuts import render
+from friendreq.models import ItemsFound, ItemRequest, STATUS_CHOICES as REQUEST_STATUS_CHOICES
+from django.http import HttpRequest, HttpResponse, Http404
+from django.shortcuts import render, redirect
+from django.contrib.auth.models import User
+from . models import CoordinatedItems
+from django.conf import settings
+from . import constants
+import sys
+import os
 
-# Create your views here.
+
+def coordinator_create_view(request):
+    if request.method == 'GET':
+        itemsToCoordinate = []
+        matchedItems = ItemsFound.objects.filter(match=True)
+        openRequests = ItemRequest.objects.filter(status=constants.request_status_open)
+        users = User.objects.all()
+
+        return render (request, 'coordinator.html', {'matchedItems':matchedItems, 'openRequests':openRequests, 'users': users})
+    else:
+        create_new_coordinations(request);
+        close_related_request(request);
+        return redirect('/coordinate')
+    
+
+def create_new_coordinations(request):
+    request_id = request.POST[constants.request_id]
+    if request.method == 'POST':
+        CoordinatedItems.objects.create(
```

Comment:
See Django's automated form management features to avoid endless variable assignment boilerplate like this:
https://docs.djangoproject.com/en/3.1/topics/forms/modelforms/

Reply:
definitely the way to go - switched to using Django forms